src/include/os_fhandle.i

```diff
@@ -156,6 +160,7 @@ static inline int
 __wt_write(WT_SESSION_IMPL *session,
     WT_FH *fh, wt_off_t offset, size_t len, const void *buf)
 {
+	struct timespec start, stop;
```

Comment:
I suppose struct timespec goes after WT_DECL_RET and other such macro types. Same holds good at other places.